src/path-watcher.js

```diff
@@ -544,12 +544,28 @@ class PathWatcherManager {
   // Private: Access the currently active manager instance, creating one if necessary.
   static active () {
     if (!this.activeManager) {
-      this.activeManager = new PathWatcherManager(atom.config.get('core.fileSystemWatcher'))
-      this.sub = atom.config.onDidChange('core.fileSystemWatcher', ({newValue}) => { this.transitionTo(newValue) })
+      const config = this.findConfig()
+
+      this.activeManager = new PathWatcherManager(config.get('core.fileSystemWatcher'))
+      this.sub = config.onDidChange('core.fileSystemWatcher', ({newValue}) => {
+        this.transitionTo(newValue)
+      })
     }
     return this.activeManager
   }
 
+  static findConfig() {
```

Comment:
There should be a space before the parentheses.
Before:
`static findConfig() {`
After:
`static findConfig () {`